Programm started at: 2025-06-30 01:27:01

Running in version 1.2[2025-06-30 23:31:36]
Git repositories synchronized
	

	NoneType: None


[2025-06-30 23:31:36]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-07-01 01:27:58

Running in version 1.2

[2025-07-01 23:31:34]
Git repositories synchronized
	

	NoneType: None


[2025-07-01 23:31:34]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-07-02 01:27:23

Running in version 1.2

[2025-07-02 23:31:59]
Git repositories synchronized
	

	NoneType: None


[2025-07-02 23:31:59]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-07-03 01:26:53

Running in version 1.2

[2025-07-03 23:31:32]
Git repositories synchronized
	

	NoneType: None


[2025-07-03 23:31:32]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-07-04 01:27:28

Running in version 1.2

[2025-07-04 14:53:54]
Error 1 at reading or storing sensor data.
	I2C read failed

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 192, in read_all
    self.read_logorithm_results()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 188, in read_logorithm_results
    raise Exception (pigpio.error_text(b))
Exception: I2C read failed


[2025-07-04 23:31:13]
Git repositories synchronized
	

	NoneType: None


[2025-07-04 23:31:13]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-07-05 01:27:08

Running in version 1.2

[2025-07-05 23:05:51]
Error 1 at reading or storing sensor data.
	'I2C read failed'

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 191, in read_all
    if self.data_available():
       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 172, in data_available
    value = self.pi.i2c_read_byte_data(self.device, CSS811_STATUS)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 2874, in i2c_read_byte_data
    return _u2i(_pigpio_command(self.sl, _PI_CMD_I2CRB, handle, reg))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 1011, in _u2i
    raise error(error_text(v))
pigpio.error: 'I2C read failed'


[2025-07-05 23:31:53]
Git repositories synchronized
	

	NoneType: None


[2025-07-05 23:31:53]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-07-06 01:26:43

Running in version 1.2

[2025-07-06 10:48:01]
Error 1 at reading or storing sensor data.
	I2C read failed

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 192, in read_all
    self.read_logorithm_results()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 188, in read_logorithm_results
    raise Exception (pigpio.error_text(b))
Exception: I2C read failed


[2025-07-06 23:31:28]
Git repositories synchronized
	

	NoneType: None


[2025-07-06 23:31:28]
Terminating programm
	

	NoneType: None


